invenio_requests/assets/semantic-ui/js/invenio_requests/requestDetails.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Tab } from "semantic-ui-react";
+import ReactDOM from "react-dom";
+
+class RequestDetails extends Component {
+  menuPanes = [
+    { menuItem: "Timeline", render: () => <Tab.Pane>TIMELINE</Tab.Pane> },
```

Comment:
Minor: I honestly prefer `Conversation` than `Timeline` as in the mocks... was this changed agreed?
```suggestion
    { menuItem: "Conversation", render: () => <Tab.Pane>TIMELINE</Tab.Pane> },
```